pkg/test/framework.go

```diff
@@ -40,36 +40,46 @@ import (
 )
 
 var (
-	// Global framework struct
+	// Global is a global framework struct that the test can use.
 	Global *Framework
 	// mutex for AddToFrameworkScheme
 	mutex = sync.Mutex{}
-	// whether to run tests in a single namespace
-	singleNamespace *bool
+	// SingleNamespace determines whether tests are to be run in a single namespace or not.
+	SingleNamespace *bool
 	// decoder used by createFromYaml
 	dynamicDecoder runtime.Decoder
 	// restMapper for the dynamic client
 	restMapper *restmapper.DeferredDiscoveryRESTMapper
 )
 
+// Framework contains all relevant variables needed for running tests with the operator-sdk.
 type Framework struct {
 	Client            *frameworkClient
 	KubeConfig        *rest.Config
 	KubeClient        kubernetes.Interface
 	Scheme            *runtime.Scheme
-	NamespacedManPath *string
+	NamespacedManPath string
```

Comment:
Was this done initially in case we could have a nil and do we not have that case anymore?

Reply:
I'm not sure why it was a pointer. We don't have any nil checks anywhere, so this doesn't affect anything. Just makes more sense to have it as a normal string IMO

Reply:
Makes sense, will this be a breaking change for the users, should we explicitly mention it in the changelog.

Reply:
I don't think it's something the users should have been accessing directly in the first place. It might make sense to unexport this field (and maybe a few others)

Reply:
Hi @AlexNPavel, 

I think that namespace can be null, is not possible the user/dev use it as an lib/imported in their project?